pilot/ARCHITECTURE.md

```diff
@@ -7,8 +7,8 @@ Istiod is structured as a modular monolith, housing a wide range of functionalit
 
 Istiod's primary role - and most code - is to dynamically configure proxies (Envoy sidecars and ingress, gRPC, ztunnel, and more). This roughly consists of 3 parts:
 1. Config ingestion (inputs to the system)
-1. Config translation
-1. Config serving (XDS)
+2. Config translation
```

Comment:
I don't get this. Isn't the number automatically formatted?

Reply:
Its numbers are equal and I want to keep it incremental.

Reply:
@yyzxw it's valid Markdown numbering method to avoid re-numbering all the following items if you insert new item in the middle.

Reply:
The numbering is by default supposed to be `1.` that is why lint is failing. This is the rule being applied here: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md029---ordered-list-item-prefix

Keep them all `1.`

Reply:
Thank you all, I've reverted.